### ISSUE TITLE
Blank screen when loading a past transaction in a workspace created with beta version

### ISSUE THREAD

**Issue by sarbogast** [trufflesuite/ganache#1167]
## Expected Behavior

I have a workspace I've been using for a while and I created with Ganache 2.0.0-beta.2. It has 2 contracts deployed and a bunch of transactions.
When I click any contract from the Contracts tab and then click one of the transactions that happened on that contract to see more details about it, I get a blank screen instead. And yesterday I did the same thing with Ganache 2.0.0-beta.2, it worked without any issue.

## Possible Solution
Maybe it is related to https://github.com/trufflesuite/ganache/issues/1155

## Steps to Reproduce (for bugs)
I don't know how to export a workspace so that you can try it for yourselves.

## Context
I'm just trying to navigate through past transactions in a contract, which is a killer feature in Ganache 2 but doesn't seem to work right now.

## Your Environment
* Version used: 2.0.0 final
* Operating System and version: MacOS 10.14.3

**Comment by sarbogast** [comment]
I was able to reproduce the same problem under Windows.
And then I thought on opening Chrome's console and found this error:
<img width="554" alt="Screenshot 2019-03-22 14 25 00" src="https://user-images.githubusercontent.com/143758/54804413-49f00d00-4cae-11e9-9667-231058802bf9.png">

**Comment by sarbogast** [comment]
And when I try an old workspace on Linux, I get an error from the get go saying:

```
Error: Could not find 'node'. NodeJS is required to be installed to link Truffle projects.
    at ChildProcess.child.on.error (/src/truffle-integration/projectDetails.js:65:17)
    at emitOne (events.js:115:13)
    at ChildProcess.emit (events.js:210:7)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:198:12)
    at onErrorNT (internal/child_process.js:374:16)
    at _combinedTickCallback (internal/process/next_tick.js:138:11)
    at process._tickCallback (internal/process/next_tick.js:180:9)
```

**Comment by sarbogast** [comment]
I created [another issue](https://github.com/trufflesuite/ganache/issues/1168) for the problem on Linux as it seemed unrelated.

**Comment by DiscRiskandBisque** [comment]
We'll try to reproduce--100% agree this is likely related to #1155. In regards to that, is this a long-running workspace and does it use automining or a set blocktime?

**Comment by DiscRiskandBisque** [comment]
#1157 could also be related.

**Comment by sarbogast** [comment]
It doesn’t use auto-mining and it sets a block time of 3 seconds. Not sure what you mean by long-running

**Comment by sarbogast** [comment]
Seems to be resolved in 2.0.1-beta.0
